plugins/bazaar/src/index.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2021 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+export { bazaarPlugin, BazaarPage } from './plugin';
+export {
+  isBazaarAvailable,
+  BazaarInfoCard,
+} from './components/BazaarComponents/BazaarInfoCard';
```

Comment:
Is the BazaarComponents folder needed? It seems a bit redundant to have plugins/**bazaar**/src/**components**/**BazaarComponents**/